SymphonyTests/ComposerTests.swift

```diff
@@ -209,6 +209,30 @@ class NavigationComposerTests: XCTestCase {
         XCTAssertNotNil(weakPushedComposable)
     }
 
+    func test_whenPresentingMultipleModals_presentingDoesNotCauseACrash() {
+
+        let presentationExpectation = expectation(description: "Second view was modally presented")
+
+        let presentedComposable1 = BasicComposable()
+        let presentedComposable2 = BasicComposable()
+
+        composer.present(presentedComposable1)
+
+        DispatchQueue.main.async {
+            self.composer.present(presentedComposable2)
```

Comment:
This /should/ work even if the second `present` call exists outside of this async call. I'd be interested in testing it that way so that usage of this feature does not depend on the developer knowing to make the second `present` call inside an async block.

I guess that seems like a stronger test; in other words, if it works when two composables are presented synchronously then it should also work when they are composed with some time between the two `present` calls.

Reply:
Tried it out and it turns out it does need the `async` call

Reply:
I wonder what's going on then. It passes consistently on my machine.

```
...
let presentedComposable1 = BasicComposable()
let presentedComposable2 = BasicComposable()

composer.present(presentedComposable1)
composer.present(presentedComposable2)

DispatchQueue.main.async {
    presentationExpectation.fulfill()
}
...
```